cmd/libs/go2idl/openapi-gen/generators/openapi.go

```diff
@@ -40,15 +40,30 @@ const tagOptional = "optional"
 
 // Known values for the tag.
 const (
-	tagValueTrue       = "true"
-	tagValueFalse      = "false"
-	tagExtensionPrefix = "x-kubernetes-"
+	tagValueTrue              = "true"
+	tagValueFalse             = "false"
+	tagExtensionPrefix        = "x-kubernetes-"
+	tagPatchStrategy          = "patchStrategy"
+	tagPatchMergeKey          = "patchMergeKey"
+	patchStrategyExtensionKey = "x-kubernetes-patch-strategy"
```

Comment:
Would it make sense to write this and below as `tagExtensionPrefix + "patch-strategy"`?